packages/toolpad-core/src/Account/Account.tsx

```diff
@@ -47,109 +65,140 @@ export interface AccountProps {
  * - [Account API](https://mui.com/toolpad/core/api/account)
  */
 function Account(props: AccountProps) {
-  const { slotProps, signInLabel = 'Sign In', signOutLabel = 'Sign Out' } = props;
-  const session = React.useContext(SessionContext);
-
+  const { slots, slotProps, signInLabel = 'Sign In', signOutLabel = 'Sign Out' } = props;
```

Comment:
How does X handle customizing their labels in e.g. the datagrid?

Reply:
The closest example I can find in the grid is customising the pagination labels https://mui.com/x/react-data-grid/pagination/#page-size-options

Reply:
https://mui.com/x/react-data-grid/localization/#locale-text

Reply:
Do you mean we should drop the `signInLabel` and `signOutLabel` props in favour of a `LabelsProvider` of some sort?

Reply:
The grid supports a `localeText` property. We could opt for following the same convention? It also reads those from the theme. We can add support for that later.